packages/web-app-files/src/fileSideBars.ts

```diff
@@ -72,12 +72,12 @@ const panelGenerators: (({
       return multipleSelection && !rootFolder
     }
   }),
-  ({ router, highlightedFile }) => ({
+  ({ highlightedFile }) => ({
     app: 'details-space-item',
     icon: 'questionnaire-line',
     title: $gettext('Details'),
     component: SpaceDetails,
-    default: isLocationSpacesActive(router, 'files-spaces-projects'),
+    default: highlightedFile?.type === 'space',
```

Comment:
Was looking at the same line in another PR yesterday and came to the conclusion that we can even have `default: true`. The panel is only enabled if the highlighted file is a space. So when it's enabled it will also always fill the role of a default panel (i.e. active as initial panel when the sidebar opens). Could as well use `default: true` then 😉

Reply:
merging it here, it still fixes something but has unnecessary code that will be fixed in https://github.com/owncloud/web/pull/7430